graph/telemetry/istio/istio.go

```diff
@@ -51,12 +50,22 @@ const (
 	tsHashMap graph.MetadataKey = "tsHashMap"
 )
 
+type waypointKey = struct {
+	Cluster   string
+	Namespace string
+	Name      string
+}
+
+// wpKey is a re-usable key struct. This works because map keys are always copies
+var wpKey = waypointKey{}
```

Comment:
If the map keys are always copied, what's the point of making this a global variable? It just has the risk of accidentally reusing state and not being threadsafe.

Reply:
Well, actually, the struct key doesn't get copied, it gets hashed. So, the struct itself is free to get changed iiuc.

Reply:
OK, pushed the change.